assignments/function-conversion.js

```diff
@@ -1,30 +1,49 @@
-// Take the commented ES5 syntax and convert it to ES6 arrow Syntax
+/// Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {
 // console.log("Function was invoked!");
 // };
 // myFunction();
 
+let myFunction = () => {console.log("Function was invoked!")};
+
+myFunction();
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+let anotherFunction = (param) => {return param;};
+
+console.log(anotherFunction("Example"));
+
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
 
+let add = (param1, param2) => {return param1 + param2;};
+
+console.log(add(1,2));
+
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
 
+let subtract = (param1, param2) => {return param1 - param2;};
+
+console.log(subtract(1,2));
 
 // Stretch
 
 // exampleArray = [1,2,3,4];
 // const triple = exampleArray.map(function (num) {
 //   return num * 3;
 // });
-// console.log(triple);
\ No newline at end of file
+// console.log(triple);
+
+const triple = (arr) => arr.map(function(num){return num *3});
```

Comment:
Good,  this `const triple = (arr) => arr.map(num => num *3 );` would give you same result